<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>echo</class>
 <widget class="QWidget" name="echo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>4</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="page0_Loading">
    <widget class="QGroupBox" name="Counting">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>请稍候...</string>
     </property>
     <widget class="QLabel" name="plan_name">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>31</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>计划循环次数:     </string>
      </property>
     </widget>
     <widget class="QLabel" name="plan_num">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="done_name">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>61</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>完成循环次数:     </string>
      </property>
     </widget>
     <widget class="QLabel" name="done_num">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page1_Manual_Input">
    <widget class="QTextEdit" name="input_textedit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="input_reset">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>146</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>重    置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="input_confirm">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>146</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>确    认</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page2_Result_Error">
    <widget class="QPushButton" name="std_show">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查看标准输出</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="test_diff">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="test_show">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查看测试输出</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="std_diff">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="input">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="input_show">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查看完整输入</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>9</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;错误信息仅显示一行，点击右侧按钮查看更多。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page3_Result_OK">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>5</pointsize>
      </font>
     </property>
     <property name="text">
      <string>测试组数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="ok_loopNum">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>45</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>5</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>5</pointsize>
      </font>
     </property>
     <property name="text">
      <string>对拍通过！！！</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>21</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>21</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
